src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1796,6 +1797,20 @@ public static void TestJsonReaderUtf8SpecialString(string jsonString, string exp
             }
         }
 
+        [Theory]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, true)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF, (byte)'1' }, false)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, true)]
+        [InlineData(new byte[] { 0xEF, 0xBB, 0xBF }, false)]
+        public static void TestBOMWithSingleJsonValue(byte[] utf8BomAndValue, bool isFinalBlock)
```

Comment:
Just for completion, can you also add a `[Theory]` by creating a `ReadOnlySequence<byte>` and passing that to the `Utf8JsonReader` as well?

You can use the testhelper to create the readonlysequence of different segment sizes (1, 2, 3):
```C#
ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(utf8BomAndValue, 1); // or 2, 3
```

Add that test to the file `Utf8JsonReaderTests.MultiSegment.cs`, for example:
https://github.com/dotnet/corefx/blob/6c95084f64df294b645729aea9934f72271292eb/src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs#L451

Reply:
Done